projects/genie.lua

```diff
@@ -87,16 +87,19 @@ function defaultConfigurations()
 	configuration "Debug"
 		targetdir(BINARY_DIR .. "Debug")
 		defines { "DEBUG" }
+		buildoptions { "-std=c++11" }
```

Comment:
I tried to compile it with MSVS, and even though ```-std=c++11``` is ignored, it causes warnings